cli/config/configuration-qemu.toml.in

```diff
@@ -121,6 +121,9 @@ virtio_fs_cache_size = @DEFVIRTIOFSCACHESIZE@
 #    Metadata, data, and pathname lookup are cached in guest and never expire.
 virtio_fs_cache = "@DEFVIRTIOFSCACHE@"
 
+# Use shared version metadata
+virtio_fs_shared_versions = @DEFVIRTIOFSSHAREDVERSIONS@
```

Comment:
@stefanha Can you elaborate more on what this option does, pros and cons of using this?

Reply:
This is an experimental feature and it's not ready for use.  I will remove it for the time being since it relies on an external daemon (ireg) that isn't managed by kata-runtime yet.  Once this feature is working it will make the cache=always mode coherent across shared volumes.

Reply:
Agreed. I think the first step is to get virtio-fs early supported with the things that are reliable in virtio-fs for now. We can definitely add the `ireg` support with a later follow up PR :)